__tests__/lib/keychain.js

```diff
@@ -7,110 +7,57 @@ import { randomBytes } from 'crypto';
  * Internal dependencies
  */
 import Insecure from '../../src/lib/keychain/insecure';
-import Browser from '../../src/lib/keychain/browser';
 
-// TODO: Random bytes
 const account = 'vip-cli-test';
 const password = randomBytes( 256 ).toString();
 
-let keychain;
-
-describe( 'token tests (secure)', () => {
-	it( 'should correctly set token', () => {
-		return keychain.setPassword( account, password ).then( async () => {
-			const passwd = keychain.getPassword( account );
-			await expect( passwd ).resolves.toBe( password );
-		} );
-	} );
-
-	it( 'should correctly delete token', () => {
-		return keychain.setPassword( account, password ).then( () => {
-			return keychain.deletePassword( account ).then( async () => {
-				const passwd = keychain.getPassword( account );
-				await expect( passwd ).resolves.toBeNull();
-			} );
-		} );
-	} );
-} );
```

Comment:
These tests are deceiving: they do not test the Secure keychain.

Jest hoists the `keychain = new Insecure( account );` statement; as a result, `keychain` is `Insecure` in these tests.

I removed these tests because the `token tests (insecure)` test suite checks these assertions.